play-json/shared/src/test/scala/JsonSharedSpec.scala

```diff
@@ -26,197 +28,201 @@ class JsonSharedSpec extends WordSpec with MustMatchers {
     (__ \ 'models).format[Map[String, String]]
   )(Car, unlift(Car.unapply))
 
+  def json[T](f: JsonFacade => T) = forAll(
```

Comment:
Update tests to ensure the behaviour coherency accross the different kind of `JsonFacade`